deploy/kubernetes/releases/csi-digitalocean-v1.1.3.yaml

```diff
@@ -0,0 +1,510 @@
+# Copyright 2019 DigitalOcean
```

Comment:
We [generate / update this file dynamically with our bump-version Makefile target](https://github.com/digitalocean/csi-digitalocean/blob/a31f090a350ed24bce07fbd34ed34ae851625fc6/Makefile#L26-L27), so I assume we should not commit it. The next release would overwrite it.

Reply:
Since I am creating a new container spec for resizer, which doesn't exist in version 1.1.2, how should we handle this? The generate script just copies the  the current version manifest tothe new version. I'll add a condition to not do that when there is a file existing for the new version?

Reply:
The way I think we can handle this is to add the resizer and apply any other changes after we have invoked `bump-version`. Essentially, the change would go in as we do the release.

Going forward, we could have a templated version of our release manifest that we can update as we add features / make changes (i.e. during development) and only populate during the release with the exact tags that we want. But I think we should create an issue for that and do it in a follow-up PR.

WDYT?

Reply:
Yeah, sounds good! I'll create the issue today. 
For this PR, I will not commit the 1.1.3 manifest, but during the new release, we should keep track of this PR and add the container spec for resizer.

Reply:
👌 I suppose we will be able to copy the relevant updates from the dev manifest that is going to be committed by this PR.